package.json

```diff
@@ -154,6 +174,39 @@
           "command": "extension.runQuery",
           "when": "editorLangId == sql"
         }
+      ],
+      "view/title": [
+        {
+          "command": "extension.addObjectExplorer",
+          "title": "%extension.addObjectExplorer%",
+          "group": "navigation"
+        }
+      ],
+      "view/item/context": [
+        {
+          "command": "extension.objectExplorerNewQuery",
+          "when": "viewItem == Server"
+        },
```

Comment:
Apparently context keys support OR now [https://github.com/microsoft/vscode/commit/893635042ccd4dbdf919a4493a062330aec2cbc7](https://github.com/microsoft/vscode/commit/893635042ccd4dbdf919a4493a062330aec2cbc7)

Reply:
Not planning on using the new or capabilities? It would be good to try them out to make sure they work. 

---
In reply to: [309266302](https://github.com/microsoft/vscode-mssql/pull/1259#discussion_r309266302) [](ancestors = 309266302)